browser/ui/webui/brave_rewards_page_ui.cc

```diff
@@ -2046,14 +2051,14 @@ void RewardsDOMHandler::GetOnboardingStatus(const base::Value::List& args) {
                          base::Value(std::move(data)));
 }
 
-void RewardsDOMHandler::SaveOnboardingResult(const base::Value::List& args) {
-  CHECK_EQ(1U, args.size());
-  if (!rewards_service_)
-    return;
-
+void RewardsDOMHandler::EnableRewards(const base::Value::List& args) {
```

Comment:
It doesn't seem like this function does anything on Android. Are we going to open the Android-native panel via a different mechanism? Or is this desktop-only for now?

Reply:
On Android, there's another mechanism to that shows a native onboarding modal when the Rewards page is accessed. I'll add some comments here to explain.